modules/swagger-codegen/src/main/resources/typescript-angular/apis.mustache

```diff
@@ -4,6 +4,10 @@
 export * from './{{ classFilename }}';
 import { {{ classname }} } from './{{ classFilename }}';
 {{/operations}}
+{{#withInterfaces}}
+export * from './{{ classFilename }}Interface'
+import { {{ classname }}Interface } from './{{ classFilename }}Interface';
+{{/withInterfaces}}
 {{/apis}}
 export const APIS = [{{#apis}}{{#operations}}{{ classname }}{{/operations}}{{^-last}}, {{/-last}}{{/apis}}];
```

Comment:
please add the generated petstore sample files. or are there none with the `with-interfaces` flag?

Reply:
You are right, the import is not necessary. Will you change that line or should I redo the pull request?

I've generated the petstore ```./bin/typescript-angular-v2-petstore-interfaces.sh```. The outcome is like expected 
```typescript
cat /Users/rainer/src/feikede/swagger-codegen/samples/client/petstore/typescript-angular-v2/with-interfaces/api/api.ts
export * from './pet.service';
import { PetService } from './pet.service';
export * from './pet.serviceInterface'
import { PetServiceInterface } from './pet.serviceInterface';
export * from './store.service';
import { StoreService } from './store.service';
export * from './store.serviceInterface'
import { StoreServiceInterface } from './store.serviceInterface';
export * from './user.service';
import { UserService } from './user.service';
export * from './user.serviceInterface'
import { UserServiceInterface } from './user.serviceInterface';
export const APIS = [PetService, StoreService, UserService];
```
But what files should I add here? The complete output as zip?
The CircleCi fails because of a joda-time download error. Not my issue.

Thanks

Reply:
you should add the `samples/client/petstore/typescript-angular-v2/with-interfaces/api/api.ts`
but anyway, I don't see why this PR is necessary, as the interfaces are exported in the interfaces file and can be imported from there, e.g. https://github.com/swagger-api/swagger-codegen/blob/master/samples/client/petstore/typescript-angular-v2/with-interfaces/api/PetApiInterface.ts#L21

Reply:
I can't remove the import, so you can remove it and simply push to your branch

Reply:
Removed the import, found that github automatically adjusts the PR :-). The petstore now looks like
```
cat /Users/rainer/src/feikede/swagger-codegen/samples/client/petstore/typescript-angular-v2/with-interfaces/api/api.ts
export * from './pet.service';
import { PetService } from './pet.service';
export * from './pet.serviceInterface'
export * from './store.service';
import { StoreService } from './store.service';
export * from './store.serviceInterface'
export * from './user.service';
import { UserService } from './user.service';
export * from './user.serviceInterface'
export const APIS = [PetService, StoreService, UserService];
```
 
Should I commit also the changed file in the petstore example? (It's only ```samples/client/petstore/typescript-angular-v2/with-interfaces/api/api.ts```)

Reply:
> Should I commit also the changed file in the petstore example? (It's only samples/client/petstore/typescript-angular-v2/with-interfaces/api/api.ts)

yes, the example files are also added and updated in the repository